crates/matrix-sdk-indexeddb/Cargo.toml

```
@@ -38,6 +38,9 @@ tokio = { workspace = true }
 tracing = { workspace = true }
 wasm-bindgen = "0.2.83"
 web-sys = { version = "0.3.57", features = ["IdbKeyRange"] }
+hkdf = "0.12.4"
+zeroize.workspace = true
```

Can we stick to one style for defining workspace deps? The one with the braces was used until now exclusively.

```suggestion
zeroize = { workspace = true }
```

---

sure, sorry, that was just what `cargo add` gave me.